.github/workflows/lint-megalinter.yaml

```diff
@@ -9,88 +6,159 @@ on:
   workflow_call:
   workflow_dispatch:
 
-env: # Comment env block if you do not want to apply fixes
+# Comment env block if you do not want to apply fixes
+env:
   # Apply linter fixes configuration
-  # APPLY_FIXES: all # When active, APPLY_FIXES must also be defined as environment variable (in github/workflows/mega-linter.yml or other CI tool)
-  APPLY_FIXES_EVENT: pull_request # Decide which event triggers application of fixes in a commit or a PR (pull_request, push, all)
-  APPLY_FIXES_MODE: commit # If APPLY_FIXES is used, defines if the fixes are directly committed (commit) or posted in a PR (pull_request)
+  #
+  # When active, APPLY_FIXES must also be defined as environment variable
+  # (in github/workflows/mega-linter.yml or other CI tool)
+  APPLY_FIXES: all
```

Comment:
I think there is a risk of conflict with modification by the pure-commit workflow if automatic modification is allowed.

Reply:
Thanks, fixed